src/functions/vacation-request/create-vacation-request/handler.ts

```diff
@@ -0,0 +1,62 @@
+import {middyfy} from "src/libs/lambda";
+import {vacationRequestService} from "src/database/services";
+import {v4 as uuidv4} from "uuid";
+import type {ValidatedEventAPIGatewayProxyEvent} from "src/libs/api-gateway";
+import type vacationRequestSchema from "src/schema/vacationRequest";
+import VacationRequestModel from "@database/models/vacationRequest";
+
+/**
+ * Handler for creating a new vacation request entry in DynamoDB.
+ *
+ * @param event - API Gateway event containing the request body.
+ * @returns Response object with status code
+ */
+export const createVacationRequestHandler: ValidatedEventAPIGatewayProxyEvent<typeof vacationRequestSchema> = async (event) => {
+    if (!event.body) {
+        return {
+            statusCode: 400,
+            body: JSON.stringify({ error: "Request body is required." })
```

Comment:
I think all error messages should be updated to refer to Vacation request, rather than Request- I think it's a bit clearer- especially if we think from a UI perspective there are many "Requests" being made

Reply:
Will do. Although I thought that "request" in this case refers to the API request, rather than to a vacation request. Doesn`t it?

Reply:
Ha you are right! Well I guess that's proof there is the possibility for confusion. So this is fine and does not need changing.